doc/apache2/Readme.md

```diff
@@ -70,7 +70,7 @@ Example config for Apache 2.4 in prefork mode:
         # Makes it possible to place your favicon at the root `web` directory in your eZ instance.
         # It will then be served directly.
         RewriteRule ^/favicon\.ico - [L]
-        RewriteRule ^/robots\.txt - [L]
+        # RewriteRule ^/robots\.txt - [L]
```

Comment:
It was suggested by the authors ([link](https://github.com/Novactive/NovaeZSEOBundle/blob/master/Resources/doc/INSTALL.md#remove-the-robotstxt-native-rewriterules-)), but if there will be no entries for `robots_disallow` in config.yml file - this change can be removed.